src/btchip_apdu_get_coin_version.c

```diff
@@ -31,19 +31,19 @@ unsigned short btchip_apdu_get_coin_version() {
         return BTCHIP_SW_CONDITIONS_OF_USE_NOT_SATISFIED;
     }
 
-    G_io_apdu_buffer[offset++] = G_coin_config->p2pkh_version >> 8;
-    G_io_apdu_buffer[offset++] = G_coin_config->p2pkh_version;
-    G_io_apdu_buffer[offset++] = G_coin_config->p2sh_version >> 8;
-    G_io_apdu_buffer[offset++] = G_coin_config->p2sh_version;
-    G_io_apdu_buffer[offset++] = G_coin_config->family;
-    G_io_apdu_buffer[offset++] = strlen(G_coin_config->coinid);
-    os_memmove(G_io_apdu_buffer + offset, G_coin_config->coinid,
-               strlen(G_coin_config->coinid));
-    offset += strlen(G_coin_config->coinid);
-    G_io_apdu_buffer[offset++] = strlen(G_coin_config->name_short);
-    os_memmove(G_io_apdu_buffer + offset, G_coin_config->name_short,
-               strlen(G_coin_config->name_short));
-    offset += strlen(G_coin_config->name_short);
+    G_io_apdu_buffer[offset++] = COIN_P2PKH_VERSION >> 8;
+    G_io_apdu_buffer[offset++] = COIN_P2PKH_VERSION;
+    G_io_apdu_buffer[offset++] = COIN_P2SH_VERSION >> 8;
+    G_io_apdu_buffer[offset++] = COIN_P2SH_VERSION;
+    G_io_apdu_buffer[offset++] = COIN_FAMILY;
+    G_io_apdu_buffer[offset++] = strlen(COIN_COINID);
```

Comment:
You can replace all `strlen(<static string>)` by `sizeof(<static string>) - 1`. This spare a very little bit of computing and flash.

Reply:
They are other places in this commit where `strlen` can be replaced.